sway-lsp/src/core/traverse_parse_tree.rs

```diff
@@ -570,33 +601,118 @@ fn handle_while_loop(body: &CodeBlock, condition: &Expression, tokens: &TokenMap
     }
 }
 
+fn literal_to_symbol_kind(value: &Literal) -> SymbolKind {
+    match value {
+        Literal::U8(..)
+        | Literal::U16(..)
+        | Literal::U32(..)
+        | Literal::U64(..)
+        | Literal::Numeric(..) => SymbolKind::NumericLiteral,
+        Literal::String(..) => SymbolKind::StringLiteral,
+        Literal::Byte(..) | Literal::B256(..) => SymbolKind::ByteLiteral,
+        Literal::Boolean(..) => SymbolKind::BoolLiteral,
+    }
+}
+
 fn collect_type_args(type_arguments: &Vec<TypeArgument>, token: &Token, tokens: &TokenMap) {
     for arg in type_arguments {
         let mut token = token.clone();
         let type_info = sway_core::type_system::look_up_type_id(arg.type_id);
+        // TODO handle tuple and arrays in type_arguments
```

Comment:
Do we have an issue open? Should link it here.

Reply:
Just added a link to the issue.